README.md

```diff
@@ -9,25 +9,43 @@ The following steps will install the `tigerbeetle-node` module to your current w
 * NodeJS >= `14.0.0`. _(If the correct version is not installed, an installation error will occur)_ 
 
 > Your operating system should be Linux (kernel >= v5.6) or macOS. Windows support is not yet available but is in the works.
-     
+
 ### YARN Package Manager
 
-```sh
-# Run the following from this directory
-yarn add
+```shell
+# Run the following from this directory:
+yarn set version latest
+yarn && yarn build && yarn add
+yarn postinstall
 ```
 or
 
-### NPM Package Manager 
-```sh
-npm install tigerbeetle-node
+### NPM Package Manager
+Run the following command from this directory:
+```shell
+npm install .
+```
+
+**Yarn - Run Test**
+Ensure TigerBeetle is running on the port configured in `test.tx`, then run: 
```

Comment:
```suggestion
Ensure TigerBeetle is running on the port configured in `test.ts`, then run: 
```
It would also be nice to give an example command to start tigerbeetle. Perhaps `yarn test` could do this automatically as we do for the benchmark?

Reply:
Good suggestion. Added.